packages/repository/src/decorators/relation.decorator.ts

```diff
@@ -5,8 +5,9 @@
 
 import {Class} from '../common-types';
 import {Entity} from '../model';
-
 import {PropertyDecoratorFactory} from '@loopback/context';
+import {property} from './model.decorator';
+import camelCase = require('lodash.camelcase');
```

Comment:
I don't see a dependency of `lodash.camelcase` in package.json.

Reply:
addressed

Reply:
addressed